packages/sdk/src/Actions/EventAction.ts

```diff
@@ -516,6 +519,29 @@ export class EventAction extends DeployableTarget<
     params: ValidateActionStepParams,
   ): Promise<Address | undefined> {
     const signature = claimant.signature;
+
+    // find message sender and return it
+    // WARNING: this is error prone in bridged transactions, as from will be exit node
```

Comment:
Maybe it should be a dev note in the jsdocs that bridging and internal transactions aren't yet supported